etc/config/contextBroker

```diff
@@ -54,3 +54,4 @@ BROKER_DATABASE_NAME=orion
 
 # Use the following variable if you need extra ops
```

Comment:
CHANGE_NEXT_RELEASE entry is missing (about that the default mode has been changed in the RPM).